### PR TITLE
fix: add Im-vishalanand

### LINKED ISSUE #1651: [PROFILE ADDITION] <Kumar Vishal Anand>
### Profile screenshot

image

### PR DESCRIPTION
**Pull request by Im-vishalanand**
resolves #1651

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/Im-vishalanand.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/108060013/211681340-3562af47-ad3f-4b9e-82b0-ba6b120ab123.png",
+    "issueId": 2701,
```
Comment (Panquesito7) [inline profiles/Im-vishalanand.json:3]: You must create a [valid issue](https://github.com/EddieHubCommunity/awesome-github-profiles/issues/new?assignees=&labels=profile+addition&template=add-an-awesome-github-profile.yml&title=%5BPROFILE+ADDITION%5D+%3Cname+here%3E) for your profile addition.

File: profiles/Im-vishalanand.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    "name": "Kumar Vishal Anand",
+    "username": "Im-vishalanand"
+}
```
Comment (Panquesito7) [inline profiles/Im-vishalanand.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by Im-vishalanand** [comment]
changes has been done. Let me know if it looks good now.

File: profiles/Im-vishalanand.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/108060013/211681340-3562af47-ad3f-4b9e-82b0-ba6b120ab123.png",
+    "issueId": 1641,
```
Comment (Panquesito7) [inline profiles/Im-vishalanand.json:3]: This is the current PR number, which won't work. An [issue](https://github.com/EddieHubCommunity/awesome-github-profiles/issues/new?assignees=&labels=profile+addition&template=add-an-awesome-github-profile.yml&title=%5BPROFILE+ADDITION%5D+%3Cname+here%3E) must be created for that.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by Im-vishalanand** [comment]
I got it. Done the changes. Hope it is good now.
Thank you so much for the guidance.

File: profiles/Im-vishalanand.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/108060013/211681340-3562af47-ad3f-4b9e-82b0-ba6b120ab123.png",
+    "issueId": 1651,
+    "name": "Kumar Vishal Anand",
+    "username": "Im-vishalanand"
+}
```
Comment (Panquesito7) [inline profiles/Im-vishalanand.json:6]: Missing an endline here.
```suggestion
}

```

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**